game/config.py

```diff
@@ -92,3 +92,6 @@
 
 # Terminal velocity
 TERMINAL_VELOCITY = 50
+
+# Generate Hills? 
+HILLSON = True
```

Comment:
To be consistent with other names, how about "HILLS_ON"?

Reply:
Good catch.